### PR TITLE
Stack

### PR DESCRIPTION
**Pull request by Sohailsaifi**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added implementation of Stack in C.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#399

### CODE REVIEW

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Sohailsaifi fine structuring one small modification, please display a message after insertion operation & create a count function to count total number of elements present at that time.

**Comment by Sohailsaifi** [comment]
> @Sohailsaifi fine structuring one small modification, please display a message after insertion operation & create a count function to count total number of elements present at that time.

Yeah Sure

File: C/ds/Stack in C.c [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        default:
+            printf("Please enter the correct choice\n");
+
+   }
+        printf("Press 1 to repeat otherwise press any key \n");
+        scanf("%d",&n);
+
+
+   }while(n == 1);
+
```
Comment (vatsalkesarwani12) [inline C/ds/Stack in C.c]: remove these redundant spaces from the entire code

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
edit readme file

**Review by siddharth25pandey — APPROVED** [review]
@Sohailsaifi Looks fine to me.
Able to  merge.
Thanks!

File: C/ds/Stack in C.c (hunk 1 above)
Comment (Sohailsaifi) [reply C/ds/Stack in C.c]: Done.

**Pull request closed without merging**